book/src/themes.md

```diff
@@ -1,38 +1,62 @@
 # Themes
 
-To use a theme add `theme = "<name>"` to your [`config.toml`](./configuration.md) at the very top of the file before the first section or select it during runtime using `:theme <name>`.
+To use a theme add `theme = "<name>"` to the top of your [`config.toml`](./configuration.md) file, or select it during runtime using `:theme <name>`.
 
-## Creating a theme
+## Creating a Theme
 
-Create a file with the name of your theme as file name (i.e `mytheme.toml`) and place it in your `themes` directory (i.e `~/.config/helix/themes`). The directory might have to be created beforehand.
+### Creating Your Theme File
 
-The names "default" and "base16_default" are reserved for the builtin themes and cannot be overridden by user defined themes.
+To create a theme file:
 
-The default theme.toml can be found [here](https://github.com/helix-editor/helix/blob/master/theme.toml), and user submitted themes [here](https://github.com/helix-editor/helix/blob/master/runtime/themes). 
+1. Create a 'themes' folder in your user configuration folder (e.g.
+   `~/.config/helix/themes`, or `%AppData%\helix\themes` on Windows).
+2. Create a file with the name of your theme as the file name (e.g.
+   `mytheme.toml`) and place it in your `themes` folder.
+
+> 💡 The names "default" and "base16_default" are reserved for built-in themes
+> and cannot be overridden by user-defined themes.
+
+### An Overview of the Theme File Format
 
 Each line in the theme file is specified as below:
 
 ```toml
 key = { fg = "#ffffff", bg = "#000000", underline = { color = "#ff0000", style = "curl"}, modifiers = ["bold", "italic"] }
 ```
 
-where `key` represents what you want to style, `fg` specifies the foreground color, `bg` the background color, `underline` the underline `style`/`color`, and `modifiers` is a list of style modifiers. `bg`, `underline` and `modifiers` can be omitted to defer to the defaults.
+Where `key` represents what you want to style, `fg` specifies the foreground color, `bg` the background color, `underline` the underline `style`/`color`, and `modifiers` is a list of style modifiers. `bg`, `underline` and `modifiers` can be omitted to defer to the defaults.
 
 To specify only the foreground color:
 
 ```toml
 key = "#ffffff"
 ```
 
-if the key contains a dot `'.'`, it must be quoted to prevent it being parsed as a [dotted key](https://toml.io/en/v1.0.0#keys).
+If the key contains a dot `'.'`, it must be quoted to prevent it being parsed as a [dotted key](https://toml.io/en/v1.0.0#keys).
 
 ```toml
 "key.key" = "#ffffff"
 ```
 
+For inspiration, you can find the default `theme.toml`
+[here](https://github.com/helix-editor/helix/blob/master/theme.toml) and
+user-submitted themes
+[here](https://github.com/helix-editor/helix/blob/master/runtime/themes).
+
+### Using the Linter
+
+If you plan to submit your theme for inclusion in Helix, it is recommended to
+use the supplied linting tool to ensure compliance with the specifications:
```

Comment:
I'm personally not a huge fan of themelint (it was merged before I had a chance to review the PR) so I'd tone this down to say something like "use the supplied linting tool to check for errors and missing scopes". It's a sanity check to find scopes you probably forgot to set (so it's useful to run even if you're not submitting the theme), but we won't be enforcing it